app/src/main/java/dev/arkbuilders/arkretouch/App.kt

```diff
@@ -1,24 +1,33 @@
-package dev.arkbuilders.arkretouch.presentation
+package dev.arkbuilders.arkretouch
 
 import android.app.Application
+import dev.arkbuilders.arkfilepicker.folders.FoldersRepo
+import dev.arkbuilders.arkretouch.di.EditModule
+import dev.arkbuilders.arkretouch.di.common.CommonModule
 import org.acra.config.dialog
 import org.acra.config.httpSender
 import org.acra.data.StringFormat
 import org.acra.ktx.initAcra
 import org.acra.sender.HttpSender
-import dev.arkbuilders.arkretouch.BuildConfig
-import dev.arkbuilders.arkretouch.R
-import dev.arkbuilders.arkfilepicker.folders.FoldersRepo
-import dev.arkbuilders.arkretouch.di.DIManager
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidLogger
+import org.koin.core.context.startKoin
+import org.koin.core.logger.Level
 import timber.log.Timber
 
 class App : Application() {
     override fun onCreate() {
         super.onCreate()
-        DIManager.init(this)
         Timber.plant(Timber.DebugTree())
         FoldersRepo.init(this)
 
+        setupKoin()
+
+        setupAcra()
+    }
+
+    // FIXME: Let's use Firebase crashlytics instead
+    private fun setupAcra() {
```

Comment:
We've shutdown ACRA already. What can we used instead of it? @hieuwu

Reply:
@mdrlzy

Reply:
Firebase or AppDynamics or something else?

Reply:
@shubertm I think ACRA is still here, please make sure to remove everything related to it